### PR TITLE
[TASK] Update some chapters

### PR DESCRIPTION
**Pull request by linawolf**
and replace code snippets with those automatically generated from the extension.

### CODE REVIEW

File: Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   	"support": {
+   		"issues": "https://github.com/TYPO3-Documentation/t3docs-screenshots/issues"
+   	},
+   	"extra": {
+   		"typo3/cms": {
+   			"extension-key": "site_package"
+   		}
+   	}
+   }
+
```
Comment (gilbertsoft) [inline Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt:39]: The minimal requirement of the core is missing here see https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ComposerJson/Index.html#minimal-composer-json

File: Documentation/CodeSnippets/TypoScript/Page.rst.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/Setup/Page.typoscript
+   :linenos:
+
+   page = PAGE
+   page {
+     typeNum = 0
+     // 10 = TEXT
```
Comment (gilbertsoft) [inline Documentation/CodeSnippets/TypoScript/Page.rst.txt:10]: Why commented?

File: Documentation/CodeSnippets/TypoScript/Part1FluidTemplateSection.rst.txt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/Setup/Part6ProcessedContent
+   :linenos:
+
+   
+   page {
+     10 {
+       dataProcessing {
+         //10 = TYPO3\CMS\Frontend\DataProcessing\MenuProcessor
```
Comment (gilbertsoft) [inline Documentation/CodeSnippets/TypoScript/Part1FluidTemplateSection.rst.txt:11]: Why commented?

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

File: Documentation/CodeSnippets/TypoScript/Part1FluidTemplateSection.rst.txt (hunk 3 above)
Comment (linawolf) [reply Documentation/CodeSnippets/TypoScript/Part1FluidTemplateSection.rst.txt:11]: The comments are used to show where in the larger context of the code we are working

File: Documentation/CodeSnippets/TypoScript/Page.rst.txt (hunk 2 above)
Comment (linawolf) [reply Documentation/CodeSnippets/TypoScript/Page.rst.txt:10]: "If you remove the comments :typoscript:`//` before line 4 and 5 there would be
an output of "Hello World!". But of course we want to output our Fluid template."

File: Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt (hunk 1 above)
Comment (linawolf) [reply Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt:39]: We would have to change that in the extension. however. I will make an issue there

Comment (linawolf) [reply Documentation/CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt:39]: https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage-Code/issues/5

**Review by susannemoog — APPROVED** [review]

**Pull request merged**